EU4toV2/Source/EU4World/World.cpp

```diff
@@ -800,11 +800,7 @@ void EU4::world::removeEmptyNations()
 	{
 		auto countryProvinces = country.second->getProvinces();
 		auto countryCores = country.second->getCores();
-		if ((countryProvinces.size() == 0) && (countryCores.size() == 0))
-		{
-			LOG(LogLevel::Debug) << "Removing empty nation " << country.first;
-		}
-		else
+		if ((countryProvinces.size() != 0) || (countryCores.size() != 0))
```

Comment:
Nit: the static analyzer I've been using on Vic2 to HoI4 (see here for details: https://www.modernescpp.com/index.php/brief-overview-of-the-pvs-studio-static-code-analyzer) would suggest 
if (!countryProvinces.empty() || !countryCores.empty())

Reply:
changed.